Upstream-Status: Accepted
Instead matchbox-window-manager-2, it is in libmatchboxwm2, which has build
failure as partial gtk-doc implementation.

Nitin A Kamble <nitin.a.kamble@intel.com> 2011/05/10
Fix following build error:
| Makefile:734: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
| ERROR: oe_runmake failed

Index: matchbox-window-manager-2/Makefile.am
===================================================================
--- matchbox-window-manager-2.orig/Makefile.am
+++ matchbox-window-manager-2/Makefile.am
@@ -3,6 +3,6 @@ SUBDIRS = matchbox data doc util
 # Extra clean files so that maintainer-clean removes *everything*
 
 snapshot:
-        $(MAKE) dist distdir=$(PACKAGE)-snapshot-`date +"%Y%m%d"`
+	$(MAKE) dist distdir=$(PACKAGE)-snapshot-`date +"%Y%m%d"`
 
 MAINTAINERCLEANFILES = aclocal.m4 compile config.guess config.sub configure depcomp install-sh ltmain.sh Makefile.in missing
